daemon.go

```diff
@@ -68,24 +69,23 @@ func (d *Daemon) InvalidateNodeToken(label string) error {
 	if err != nil {
 		return err
 	}
-	node.ClearToken()
-	return nil
+	return node.ClearToken()
 }
 
 // Get the node with the specified label, and check that `token` is valid for it.
 // Returns an error if the node does not exist or token is invalid.
-func (d *Daemon) getNodeWithToken(label string, token *Token) (*Node, error) {
+func (d *Daemon) getNodeWithToken(label string, tok *token.Token) (*Node, error) {
 	node, err := d.state.GetNode(label)
 	if err != nil {
 		return nil, err
 	}
-	if !node.ValidToken(*token) {
-		return nil, ErrInvalidToken
+	if !node.ValidToken(*tok) {
+		return nil, token.ErrInvalidToken
 	}
 	return node, nil
 }
 
-func (d *Daemon) usingNodeWithToken(label string, token *Token,
+func (d *Daemon) usingNodeWithToken(label string, token *token.Token,
```

Comment:
dumb golang question: the variable name `token` here is shadowing the import name `token`, wouldn't that be a problem? (you are doing this in several other places too, except in `getNodeWithToken`)

Reply:
It doesn't break anything here because we don't try to reference the package within the scope of this variable (which is why I missed it when doing the replacement). But I think just for clarity's sake I think I'll rename it to `tok` here as well.

Reply:
Update: did the renaming more thoroughly